CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## edge-19.5.2
+
+* CLI
+  * Fixed `linkerd check` and `linkerd dashboard` failing when any control plane
+    pod is not ready, even when multiple replicas exist (as in HA mode)
+* Controller
+  * Fixed control plane components failing on startup when the Kubernetes API
+    returns an `ErrGroupDiscoveryFailed`
+* Proxy
+  * Added a dispatch timeout that bounds the amount of time a request can be
```

Comment:
TIOLI: "limits the amount of time" seems easier to understand to me instead of "bounds"